docs/eventing/samples/sequence/sequence-with-broker-trigger/README.md

```diff
@@ -1,14 +1,15 @@
 We are going to create the following logical configuration. We create a
```

Comment:
Please fix all the places where this doc says "we" - https://developers.google.com/style/person

Reply:
This PR is only about updating the samples to use the V1 API. No additional text has been written. Can we merge this PR before the 0.16 doc release?